W xsec improvement: Trigger Efficiency

lmcairns-mphys opened the pull request:
@mvesteri 
Trigger efficiency has been included in the W xsec analysis. See report: [measurement_report.pdf](https://github.com/r-preston/MPhysProject2021/files/6023220/measurement_report.pdf).

I had a question regarding the error propagation of trigger efficiency in the ratios. As we are using the same value for each cross section, the trigger efficiency cancels out of the ratios, the same as luminosity, and hence would not appear in the ratio errors. However in each of the LHCb papers I have read, the error due to trigger efficiency is included in the ratio error propagation. Why is this the case?

lmcairns-mphys committed:
Inclusion of trigger efficiency in W xsec calculations

lmcairns-mphys committed:
Update W xsec latex formatting to include trig eff error

lmcairns-mphys committed:
Update report to include trigger efficiency in calculations and workflow

lmcairns-mphys committed:
Update trigger eff latec output

mvesteri commented:
In the LHCb publication it will be more complicated because the efficiencies will have been determined and corrected for in bins of the muon kinematics. We are just assuming that the trigger efficiency is independent of the muon kinematics. However, the efficiency uncertainty still won't cancel completely because the correction is applied differently to the W and Z selections. For the W it should be very simple. For the Z it is slightly more complicated because we require that *either* muon is "TOS".

lmcairns-mphys committed:
Update to trigger efficiency calculation: use fiducial regions

lmcairns-mphys committed:
Update to W xsec + ratio calculation: use correct trig effs, and corrected error propagation

lmcairns-mphys committed:
Update to Z xsec calculation: corrected error propagation

lmcairns-mphys committed:
Updated W latex formatting: include efficiency errors in latex output

lmcairns-mphys committed:
Updated report: includes corrected error propagation and more detail on trig eff

lmcairns-mphys commented:
@mvesteri 
[measurement_report.pdf](https://github.com/r-preston/MPhysProject2021/files/6031019/measurement_report.pdf)
In this update I have:
- Corrected W xsec calculations to use the correct versions of trigger efficiency
- Updated the trigger efficiency calculation to only count TOS in the fiducial region
- Corrected error propagation. I had been incorrectly propagating errors on the cross sections, which has now been corrected, giving the correct values for the separated errors.
- Added full error propagation for the WW and WZ ratios, giving separated statistical and efficiency errors.
- Written up these full calculations and error propagation in the latex report (seen in Sections 2.3, 2.4, and 4).

The new results are given in the report.

mvesteri commented:
This looks great @lmcairns
